lib/chef/knife/vsphere_datastore_list.rb

```diff
@@ -18,23 +18,6 @@
 require 'chef/knife'
 require 'chef/knife/base_vsphere_command'
 
-def number_to_human_size(number)
```

Comment:
Did you move this to the base class? I see that the base class calls it but doesn't define it, though it's defined in a couple of other classes.